DEPS

```diff
@@ -53,6 +53,9 @@ deps = {
    # build. Eventually, we'll manage these dependencies together with Fuchsia
    # and not have to specific specific hashes.
 
+  'src/apps/mozart':
+   Var('fuchsia_git') + '/mozart' + '@' + '496cc8a179d06b74f37c08382efe139831b0f0a5',
```

Comment:
I'd like to avoid pulling in mozart via DEPS.  Instead, we should isolate the code that's tied into Fuchsia into build targets that build only for Fuchsia.